ee/tabby-webserver/src/service/analytic.rs

```diff
@@ -39,13 +39,23 @@ impl AnalyticService for AnalyticServiceImpl {
         start: DateTime<Utc>,
         end: DateTime<Utc>,
         users: Vec<ID>,
-        languages: Vec<Language>,
+        mut languages: Vec<Language>,
     ) -> Result<Vec<CompletionStats>> {
         let users = convert_ids(users);
-        let languages = languages.into_iter().map(|l| l.to_string()).collect();
+
+        let not_languages = languages
```

Comment:
```suggestion
        let all_languages = Language::all_known()
```

Other means languages not in enum.